src/models.py

```diff
@@ -0,0 +1,27 @@
+import torch
+from ultralytics import YOLO
+import torch.nn as nn
+
+class AxonSegmentationModel(YOLO):
+    def __init__(self, model_path=None):
+        super().__init__(model_path)
+
+        self.input_channels = 3  # Change if your preprocessed images have different channels (e.g., grayscale)
```

Comment:
Aren't our images grayscale?

Reply:
Yes, finally I did not use this file for the yolo training, it was simply a template if future implementaiton is required. I will update the PR to remove unecessary stuff